docs/docsite/rst/playbook_guide/playbooks_loops.rst

```diff
@@ -474,6 +436,117 @@ For role authors, writing roles that allow loops, instead of dictating the requi
 
     "{{ lookup('vars', ansible_loop_var) }}"
 
+
+.. _nested_loops:
+
+Nested Loops
+============
+
+While we are using ``loop`` in these examples, the same applies to ``with_<lookup>``.
+
+Iterating over nested lists
+---------------------------
+
+The simplest way to 'nest' loops is to avoid nesting loops, just format the data to achieve the same result.
+You can use Jinja2 expressions to iterate over complex lists. For example, a loop can combine nested lists, which simulates a nested loop.
+
+.. code-block:: yaml+jinja
+
+    - name: Give users access to multiple databases
+      community.mysql.mysql_user:
+        name: "{{ item[0] }}"
+        priv: "{{ item[1] }}.*:ALL"
+        append_privs: true
+        password: "foo"
+      loop: "{{ ['alice', 'bob'] | product(['clientdb', 'employeedb', 'providerdb']) | list }}"
+
+Stacking loops via include_tasks
+--------------------------------
+.. versionadded:: 2.1
+
+You can nest two looping tasks using ``include_tasks``. However, by default, Ansible sets the loop variable ``item`` for each loop.
+This means the inner, nested loop will overwrite the value of ``item`` from the outer loop.
+To avoid this, you can specify the name of the variable for each loop using ``loop_var`` with ``loop_control``.
+
+.. code-block:: yaml+jinja
+
+    # main.yml
+    - include_task: inner.yml
+      loop:
+        - 1
+        - 2
+        - 3
+      loop_control:
+        loop_var: outer_item
+
+    # inner.yml
+    - name: Print outer and inner items
+      ansible.builtin.debug:
+        msg: "outer item={{ outer_item }} inner item={{ item }}"
+      loop:
+        - a
+        - b
+        - c
+
+.. note:: If Ansible detects that the current loop is using a variable that has already been defined, it will raise an error to fail the task.
+
+Until and loop
+--------------
+
+The ``until`` condition will apply per ``item`` of the ``loop``:
+
+.. code-block:: yaml+jinja
+
+    - debug: msg={{item}}
+      loop:
+        - 1
+        - 2
+        - 3
+      retries: 2
+      until: item > 2
+
+This will make Ansible retry the first 2 items and extre 2 times, then fail the item on the 3rd attempt,
+then succeed at the first attempt on the 3rd item, in the end failing the task as a whole.
+
+.. code-block:: none
```

Comment:
> .. code-block:: none

Don't we have a special output type for playbook output?

Reply:
I have not found it, looked at other examples, that is where i got `none` from